De novo repeat annotation documnet.tex

```diff
@@ -1,321 +1,508 @@
+		\huge \textbf{CARP Document}
+	\end{center}
+	
+	% Insert table of contents
+	\tableofcontents
+	\newpage
+	
+	\pagebreak
+	
+	%------------------------------------------------------------%
+	%------------------------- Biogo Document -------------------%
+	%------------------------------------------------------------%
+	
+	
+	% CARP documentation
+	\begin{center}
+		\section{CARP manual 1.1}
+		\noindent\textbf{b\'iogo written by: Dan Kortschak} \\
+		\textbf{Document organised by: Lu Zeng} \\
+		\today
+	\end{center}
+	
+	\noindent krishna and igor are \textit{ab initio} repeat family identification and annotation programs, that  identify repeat element boundaries and family relationships from whole-genome sequence data. These programs build, refine and classify consensus models of putative interspersed repeats. krishna and igor are built using b\'iogo \href{<url>}(\url{https://github.com/biogo/biogo/}), a bioinformatics library for the Go language. 
+	
+	\noindent \textbf{Disclaimer} This document is provided to assist researchers with linux command line experience.  We have done our best to provide usable instructions, examples and advice, but users assume full responsibility for the output they generate and the authors accept no responsibility for user generated output from any programs or methods listed herein.  
+	
+	%------------------------------------------------------------------
+	% Load prerequisite packages 
+	%------------------------------------------------------------------
+	
+	\subsection{{Prerequisites}}
+	
+	\subsubsection{ Download Go }
+	Available at \href{<url>} (\url{https://golang.org/dl}).\\
+	For installation details, follow the instructions on the \href{https://golang.org/doc/install#install} {\color{blue}Go installation} page. 
+	
+	\subsubsection{Git}
+	To perform the next step you will need Git to be installed. (Check that you have a git command before proceeding.) \\
+	
+	\noindent If you do not have a working Git installation, follow the instructions on the \href{https://git-scm.com/downloads}{\color{blue}Git download} page.
+	
+	\subsubsection{Download b\'iogo Packages}
+	{\color{red} Note:}  For convenience, add the workspace's bin subdirectory to your PATH, or add in \$HOME/.profile. 
+	\begin{lstlisting}[language=bash]
+	export PATH=$PATH:$(go env GOPATH)/bin
+	\end{lstlisting}
+	\noindent Download and install krishna and igor packages from github. 
+	
+	\begin{lstlisting}[language=bash, columns=flexible]
+	go get -u github.com/biogo/examples/krishna
+	go get github.com/biogo/examples/krishna/matrix
+	go get github.com/biogo/examples/igor
+	go get github.com/biogo/examples/igor/seqer
+	go get github.com/biogo/examples/igor/gffer
+	\end{lstlisting}
+	
+	\subsubsection{ Install CENSOR}
+	Install censor to screen target genomes against a reference collection of repeats with masking symbols, as well as generating a report classifying all repeats found. CENSOR needs wu-blast/NCBI-blast and Bioperl installed.\\
```

Comment:
Point to links for downloading wu-blast (https://www.advbiocomp.com/blast/obsolete/)/NCBI blast and Bioperl.

Reply:
Done